app/js/main.js

```diff
@@ -0,0 +1,119 @@
+const items = document.getElementsByClassName('item');
+const paras = document.getElementsByTagName('p');
+
+const HOST = 'https://bdteamfive.herokuapp.com';
+// const HOST = 'http://localhost:80';
+
+const getProducts = async () => {
+  const result = await fetch(`${HOST}/products`);
+  const json = await result.json();
+  console.log(json);
+}
+
+const setHeight = () => {
+  for (let item of items) {
+    item.style.height = item.offsetWidth + 'px';
+  }
+}
+
+setHeight()
+window.addEventListener('resize', () => {
+  setHeight();
+})

```

Comment:
You need eslint and prettier
```suggestion
setHeight();
window.addEventListener('resize', () => {
  setHeight();
});
```